<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapListSelector</class>
 <widget class="QDialog" name="MapListSelector">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>257</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Doomseeker - Select Maps</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="descriptionSelect">
     <property name="text">
      <string>All selected maps will get added to the map list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listMapsWidget">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLoading">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reading wads...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonHLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="InterpolateCheckBox" name="checkAll">
       <property name="text">
        <string>Select All</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../media/resources.qrc">
         <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InterpolateCheckBox</class>
   <extends>QCheckBox</extends>
   <header>gui/widgets/interpolatecheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../media/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>MapListSelector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MapListSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAll</sender>
   <signal>clicked()</signal>
   <receiver>MapListSelector</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listMapsWidget</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>MapListSelector</receiver>
   <slot>listItemChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listMapsWidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MapListSelector</receiver>
   <slot>showContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectAll()</slot>
  <slot>listItemChanged()</slot>
  <slot>showContextMenu(QPoint)</slot>
 </slots>
</ui>
